### PR TITLE
chore(network): Updated installpath to reflect nerdlets + docs option

### PR DESCRIPTION
**Pull request by nr-jbiggley**
# Summary

Updated thse packs to reflect the preferred option to use the respective nerdlet with a fallback of the docs site.

### CODE REVIEW

File: install/network/flow/install.yml [hunk 1]
```diff
@@ -5,10 +5,17 @@ description: |
...
-  destination: cloud
+  type: agent
+  destination: host
+  os:
+    - linux
 
 install:
+  mode: nerdlet
+  destination:
+    nerdletId: network-performance-monitoring.setup-network-flow
```
Comment (polfliet) [inline install/network/flow/install.yml:16]: @kidk @jbeveland27 is calling this Nerdlet already working?

Comment (polfliet) [reply install/network/flow/install.yml:16]: ok, exactly we need to hold off merging this PR then
@nr-jbiggley can you work with the Justin and the Kentik product team who build that install nerdlet?

Comment (nr-jbiggley) [reply install/network/flow/install.yml:16]: Already in progress @polfliet

Comment (polfliet) [reply install/network/flow/install.yml:16]: bam!

**Comment by nr-jbiggley** [comment]
Replaced with [PR 480](https://github.com/newrelic/newrelic-quickstarts/pull/480) for consistency purposes.

**Pull request closed without merging**